Lib/glyphsLib/builder/axes.py

```diff
@@ -17,24 +17,7 @@
 
 from glyphsLib import classes
 from glyphsLib.classes import WEIGHT_CODES, WIDTH_CODES
-from .constants import GLYPHLIB_PREFIX
-
-# This is a key into GSFont.userData to store axes defined in the designspace
-AXES_KEY = GLYPHLIB_PREFIX + "axes"
```

Comment:
was this AXES_KEY unused? looks like you removed it

Reply:
Yeah, removed unused constant in e1d1e2e3. I've been running the test suite between commits and it works.